src/Event/MultiversionManagerEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drupal\multiversion\Event;
+
+use Drupal\Core\Entity\ContentEntityTypeInterface;
+use Drupal\multiversion\MultiversionMigrationInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * MultiversionManagerEvent class.
+ */
+class MultiversionManagerEvent extends Event {
+
+  /**
+   * @var \Drupal\Core\Entity\ContentEntityTypeInterface[]
+   */
+  protected $entityTypes;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct($entity_types, MultiversionMigrationInterface $migration) {
```

Comment:
Second parameter is not used and the parent class doesn't have a `__construct` method to use {@inheritdoc} annotation.